packages/common/src/types.ts

```diff
@@ -59,6 +59,7 @@ export interface PDFRenderProps<T extends Schema> {
  * @property {string} key - The key of the schema object.
  * @property {string} value - The string used for UI rendering.
  * @property {(value: string) => void} [onChange] - Used to change the value. Only applicable when the mode is 'form' or 'designer'.
+ * @property {(key: string, value: string) => void} [onChange] - Used to change another attribute for the schema. Only applicable when the mode is 'form' or 'designer'.
```

Comment:
onChange → onCustomAttributeChange

 * @property {(key: string, value: string) => void} [onCustomAttributeChange] - Used to change another attribute for the schema. Only applicable when the mode is 'form' or 'designer'.